CRM/Core/BAO/CustomGroup.php

```diff
@@ -128,7 +128,7 @@ public static function create(&$params) {
       $fields[] = 'is_public';
     }
     foreach ($fields as $field) {
-      if (isset($params[$field]) || $field == 'is_multiple') {
+      if (isset($params[$field])) {
         $group->$field = CRM_Utils_Array::value($field, $params, FALSE);
```

Comment:
@mickadoo we can do further cleanup here -- we're inside an `if` statement so we know the value is set already. So there's no reason to use `CRM_Utils_Array::value()` which safeguards against it not being set.

Reply:
Good point, updated